helm/oauth2-proxy/Chart.yaml

```diff
@@ -14,7 +14,7 @@ keywords:
   - redis
 dependencies:
   - name: redis
-    version: ~10.6.0
+    version: ~16.3.1
```

Comment:
https://github.com/bitnami/charts/blob/master/bitnami/redis/Chart.yaml#L27 16.4.0 is out 😝